packages/forms/resources/js/components/select.js

```diff
@@ -39,6 +40,8 @@ export default function selectFormComponent({
         state,
 
         init: async function () {
+            console.log(isSearchable)
```

Comment:
Make sure to remove this next time. 😉 I've done that now.

Reply:
Btw, really like this small PR. 😄

Reply:
LOL im sorry

Reply:
that was me